tests/testthat/test-stat_labels.R

```diff
@@ -0,0 +1,124 @@
+test_that("ard_continuous() labels", {
+
+  # formula
+  ard_test <- ard_continuous(data = ADSL,
+                             by = "ARM",
+                             variables = c("AGE","BMIBL"),
+                             stat_labels = everything() ~ list(c("min","max") ~ "min - max"))
+
+  ard_test_labs <- ard_test %>%
+    dplyr::filter(stat_name %in% c("min","max")) |>
+    dplyr::pull(stat_label) |>
+    unique()
+
+  expect_equal(ard_test_labs,
+               "min - max")
+
+  # list
+  ard_test <- ard_continuous(data = ADSL,
+                             by = "ARM",
+                             variables = c("AGE","BMIBL"),
+                             stat_labels = everything() ~ list(p25 = "25th %ile", p75 = "75th %ile"))
+
+  actual_labs <- ard_test %>%
```

Comment:
Do you mind updating the tests where we create a tibble to match against to snapshot tests? I think they are easier to maintain in the future.